### ISSUE TITLE
Add `Selenium` "Edge in IE mode" FAQ

### ISSUE THREAD

**Issue by cmin764** [robocorp/rpaframework#1076]
Add a troubleshooting section in the ~main docstring~ docs given the `Selenium` library for all the headaches (and their solutions) you may encounter when automating a legacy web app in this mode:
- 32bit vs. 64bit on the webdriver used.
- How to pass custom webdriver or place one in PATH which will take precedence. (and not trying to download one)
- [Enhanced] protected mode disabling/enabling to allow the automation perform nicely.
- How to enable private mode and its quirks.
- Registry editing for preserving the connection with the webdriver safe.
- Beating the stack protection given buffer overrun errors. (OS configuration)
- Waiting on window handles prior to switching to one that just appeared.
- Link template example given this: [Selenium automation with Edge in Internet Explorer mode](https://robocorp.com/portal/robot/robocorp/example-ie-mode-edge)
- Firewall allowance whitelist and SSL disabling instructions. (when having issues with webdriver downloading)

**Comment by cmin764** [comment]
Link somewhere this official [article](https://learn.microsoft.com/en-us/microsoft-edge/webdriver-chromium/ie-mode?tabs=python).

**Comment by cmin764** [comment]
Closing this in favor to: https://github.com/robocorp/documentation/issues/1022
